config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits.yaml

```diff
@@ -116,7 +117,7 @@ presubmits:
       - image: gcr.io/k8s-testimages/kubekins-e2e:v20200831-9529f9c-1.18
         args:
           - runner.sh
-          - "./scripts/ci-e2e.sh"
+          - "./scripts/ci-e2e.sh smoke"
```

Comment:
Instead of adding conditional logic to ci-e2e.sh, why not set the GINKGO_FOCUS environment variable?

Reply:
+1 Also we should start to introduce labels to the test name, like [Conformance], [Slow], and eventually something CAPI specific like [Upgrades], [SelfHosting], ... so we can choose what fits in the smoke test by label instead of by name

Reply:
@sedefsavas I believe this is what @randomvariable is suggesting, https://github.com/kubernetes/test-infra/blob/e6b25da2c1bc8057d51e80828af0cfe4e08a022d/config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits.yaml#L169-L178

with `GINKGO_FOCUS` thus not having to make changes to our ci-e2e.sh script as well.